CONTRIBUTING.md

```diff
@@ -77,9 +77,9 @@ When adding new code make sure to cover the following mandatory points:
 
 ### Naming Convention For Branches
 
-When creating a branch, please follow the naming convention: 
+When creating a branch, please follow the naming convention:
 
-Format: `<type>/#<issueNumber>-<name>`
```

Comment:
I think the # sign was required for github to automatically create a link to the issue

Reply:
resolved as it is not valid symbol for branch names